qiskit_jku_provider/qasm_simulator_jku.py

```diff
@@ -23,13 +23,16 @@
 from collections import OrderedDict, Counter
 import numpy as np
 
-from qiskit.backends import BaseBackend
+#from qiskit.backends import BaseBackend
 from qiskit.backends.aer import AerJob
```

Comment:
Can you replace this import, in order not to depend on `aer`? Ideally using your own `Job` class (similar to https://github.com/Qiskit/qiskit-projectq-provider/blob/master/qiskit_addon_projectq/projectqjob.py).

Reply:
Done, thanks.